frontend/src/lib/logic/apiStatusLogic.ts

```diff
@@ -46,6 +61,13 @@ export const apiStatusLogic = kea<apiStatusLogicType>([
                     const data = await response?.json()
                     if (data.detail === 'This action requires you to be recently authenticated.') {
                         actions.setTimeSensitiveAuthenticationRequired(true)
+                    } else if (data.code === 'permission_denied') {
+                        // TODO - only do if the RBAC feature flag is enabled
```

Comment:
importing feature flag logic here is breaking this logic so haven't figured out this check yet